assets/javascripts/running.js

```diff
@@ -268,4 +269,45 @@ function pauseLiveView() {
   });
 }
 
+function setupIsotovideoProxyConnection() {
```

Comment:
This function conflicts with the one I created here: https://github.com/os-autoinst/openQA/pull/1691

Since this is only C&P from the web socket console, I'd propose to use my version. It already considers that sometimes one has to use the status-only route and sometimes the developer-session route.